### PR TITLE
Fixed bounding box

### PR DESCRIPTION
**Pull request by nnmm**
* Allow choosing a fixed bbox that encompasses the earth instead of determining the box from data
* The bbox dimensions were set to a number of meters such that the earth can fit in the bounding box

### CODE REVIEW

**Comment by nnmm** [comment]
ptal @feuerste

File: src/bin/build_octree.rs [hunk 1]
```diff
@@ -29,10 +29,24 @@ struct CommandlineArguments {
     /// This decides on the number of bits used to encode each node.
     #[structopt(long = "resolution", default_value = "0.001")]
     resolution: f64,
+    // Flag to skip bbox calculation and use a fixed bbox.
+    // The fixed bbox is extends 6400000 length units in each direction.
```
Comment (feuerste) [inline src/bin/build_octree.rs]: ... -> `The fixed bbox extents have 6_400_000 length units in each direction, so they cover the whole earth when using ECEF coordinates.`?

**Comment by nnmm** [comment]
@wally-the-cartographer merge

**Comment by tblaette** [comment]
@wally-the-cartographer merge

**Comment by wally-the-cartographer** [comment]
Merge requested by authorized user tblaette. Merge queue now has a length of 1.

**Pull request merged**